src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  const error = 'Bzzz... Error!';
+
+  return (a) => {
+    count++;
```

Comment:
I wonder whether we should increment `count` in this place. Right now, the count will be incremented even if we don't invoke the last returned function.

I would put it in the last function, but it is not specified in the task description so I leave the decision to you. I just wanted to make sure that you are aware of this closure behavior.